### ISSUE TITLE
Shouldn't the per-project config file (/path/to/my/project/.npmrc) be in the .gitignore file

### ISSUE THREAD

**Issue by cquezel** [flexdinesh/npm-module-boilerplate#2]
Shouldn't the per-project config file (/path/to/my/project/.npmrc) be in the .gitignore file?

Ref: https://docs.npmjs.com/files/npmrc.html

**Comment by flexdinesh** [comment]
@cquezel But we don't have `.npmrc` in our project. If you think it's important, would you like to create a PR on why we need that and add the file and it's entry in `.gitignore`?

**Comment by cquezel** [comment]
I thought this was a "general" project for starting a npm module. The .npmrc file is a user per project configuration file used by npm. I think that most users would not want their personal configuration to be checked in to GIT or to use someone else's configuration which was accidentally checked in.

**Comment by flexdinesh** [comment]
You're right. I agree that this project should hightlight the use of `.npmrc` file. Feel free to create a PR or I can get to it in the next few days.
